fabtests/scripts/runfabtests.py

```diff
@@ -187,8 +200,10 @@ def fabtests_args_to_pytest_args(fabtests_args, shared_options):
 
     if fabtests_args.junit_xml:
         pytest_args.append("--junit-xml")
-        pytest_args.append(os.path.abspath(fabtests_args.junit_xml))
-        pytest_args.append("--self-contained-html")
```

Comment:
why this arg is dropped?

Reply:
It seems that this should be added to the `fabtests_args.html` path.

Reply:
It was not needed for xml. 

It is already in `fabtests.html` path.